Translations/pt_BR.po

```diff
@@ -1568,128 +1559,132 @@ msgstr "Pausa"
 
 #: Source/help.cpp:26
 msgid "$Keyboard Shortcuts:"
-msgstr ""
+msgstr "$Atalhos de teclado:"
 
 #: Source/help.cpp:27
 msgid "F1:    Open Help Screen"
-msgstr ""
+msgstr "F1:    Abrir tela de ajuda"
 
 #: Source/help.cpp:28
 msgid "Esc:   Display Main Menu"
-msgstr ""
+msgstr "Esc:   Mostrar menu principal"
 
 #: Source/help.cpp:29
 msgid "Tab:   Display Auto-map"
-msgstr ""
+msgstr "Tab:   Mostrar automapa"
 
 #: Source/help.cpp:30
 msgid "Space: Hide all info screens"
-msgstr ""
+msgstr "Espaço: Ocultar todas as janelas de informação"
 
 #: Source/help.cpp:31
 msgid "S: Open Speedbook"
-msgstr ""
+msgstr "S: Abrir livro da velocidade"
 
 #: Source/help.cpp:32
-#, fuzzy
-#| msgid "Spell book"
 msgid "B: Open Spellbook"
-msgstr "Livro de magias"
+msgstr "B: Abrir livro de feitiços"
 
 #: Source/help.cpp:33
 msgid "I: Open Inventory screen"
-msgstr ""
+msgstr "I: Abrir tela de inventário"
 
 #: Source/help.cpp:34
 msgid "C: Open Character screen"
-msgstr ""
+msgstr "C: Abrir tela de personagem"
 
 #: Source/help.cpp:35
-#, fuzzy
-#| msgid "Quests log"
 msgid "Q: Open Quest log"
-msgstr "Registro de missões"
+msgstr "Q: Abrir registro de missões"
 
 #: Source/help.cpp:36
 msgid "F: Reduce screen brightness"
-msgstr ""
+msgstr "F: Reduzir o brilho da tela"
 
 #: Source/help.cpp:37
-#, fuzzy
-#| msgid "increase strength"
 msgid "G: Increase screen brightness"
-msgstr "aumenta força"
+msgstr "G: Aumentar o brilho da tela"
 
 #: Source/help.cpp:38
 msgid "Z: Zoom Game Screen"
-msgstr ""
+msgstr "Z: Zoom da tela do jogo"
 
 #: Source/help.cpp:39
 msgid "+ / -: Zoom Automap"
-msgstr ""
+msgstr "+ / -: Zoom automático"
 
 #: Source/help.cpp:40
 msgid "1 - 8: Use Belt item"
-msgstr ""
+msgstr "1 - 8: Usar item de cinto"
 
 #: Source/help.cpp:41
 msgid "F5, F6, F7, F8:     Set hotkey for skill or spell"
 msgstr ""
+"F5, F6, F7, F8:     Estabelecer tecla de atalho para habilidade ou feitiço"
 
 #: Source/help.cpp:42
 msgid "Shift + Left Mouse Button: Attack without moving"
-msgstr ""
+msgstr "Shift + clique esquerdo: Atacar sem se mover"
 
 #: Source/help.cpp:43
 msgid "Shift + Left Mouse Button (on character screen): Assign all stat points"
 msgstr ""
+"Shift + clique esquerdo (na tela de personagem): Atribuir todos os pontos de "
+"estatística"
```

Comment:
I personally think `stat point` can be translated to `atributo`, this is the default translation from Dungeon & Dragons.

```suggestion
"atributos"
```